app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(8, "bottom");
+
+  clear: both;
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+
+  .govuk-govspeak {
+    p:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .govuk-body:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-intervention__title {
+  @include govuk-font(24, $weight: bold);
+  margin-top: 0;
+  @include govuk-responsive-margin(4, "bottom");
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  a {
+    @include govuk-link-common;
+    @include govuk-link-style-default;
+    @include govuk-link-print-friendly;
+  }
```

Comment:
If the content is hardcoded at the moment, is it worth removing the `a` styles here and using `govuk-link` in the HTML?